docs/tf_model_binary_input.md

```diff
@@ -35,32 +35,35 @@ docker run -u $(id -u):$(id -g) -v ${PWD}/resnet_v2/:/resnet openvino/ubuntu20_d
 *Note:* You can find out more about [TensorFlow Model conversion into Intermediate Representation](https://docs.openvino.ai/2022.1/openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_TensorFlow.html) if your model is stored in other formats.
 
 This operation will create model files in `${PWD}/resnet_v2/models/resnet/1/` folder.
-```Bash
-saved_model.bin
-saved_model.mapping
-saved_model.xml
+```bash
+tree resnet_v2/models/resnet/1
+resnet_v2/models/resnet/1
```

Comment:
Command and command output are in the same bash bracket, will it work?

Reply:
Yes, I have a list of executable bash commands and every line that doesn't match it is treated as expected output